driver-adapters/neon-node-basic/prisma/schema.prisma

```diff
@@ -0,0 +1,16 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  // fake url to ensure the driver adapter is used
+  url      = "postgresql://prisma:prisma@localhost:5432/db?schema=public"
```

Comment:
We could make it clearer that it's a fake url in the url itself, I think

Reply:
`postgresql://foo` usually works

Reply:
Replaced all the urls with that